src/Sulu/Bundle/AdminBundle/Resources/js/containers/Toolbar/DefaultButton.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import Icon from '../../components/Icon';
+import type {DefaultButtonType} from './types';
+import React from 'react';
+import buttonStyles from './button.scss'; 
+import classNames from 'classnames';
+
+const ICON_ARROW_DOWN = 'chevron-down';
+
+export default class DefaultButton extends React.PureComponent {
+    props: DefaultButtonType;
+
+    static defaultProps = {
+        disabled: false,
+        hasOptions: false,
```

Comment:
I think `trailingIcon` is already the cause of that option. I would keep `hasOptions`, `hasChildren` or something similar. If we change the behavior of the component to show something different later, we don't have to rename that option.